src/main/java/no/nav/fo/veilarbregistrering/bruker/adapter/VeilArbPersonClient.java

```diff
@@ -44,13 +41,10 @@
             return Optional.empty();
 
         } catch (ForbiddenException e) {
-            LOG.warn("Bruker har ikke tilgang på å hente ut geografisk tilknytning fra VeilArbPerson-tjenesten.", e);
-            Response response = e.getResponse();
-            throw new WebApplicationException(response);
+            throw new RuntimeException("Bruker har ikke tilgang på å hente ut geografisk tilknytning fra VeilArbPerson-tjenesten.", e);
```

Comment:
Siden vi håndterer disse exception internt foreløpig, ser jeg ingen grunn til å gjøre det vanskeligere for oss selv.
Vi kaster en runtime med feilemelding, og fanger og logger i neste steg.